doc/guides/porting/arch.rst

```diff
@@ -441,13 +441,19 @@ However, a real implementation is strongly recommended.
 Fault Management
 ****************
 
-Each architecture provides two fatal error handlers:
-
-* :code:`_NanoFatalErrorHandler`, called by software for unrecoverable errors.
-* :code:`_SysFatalErrorHandler`, which makes the decision on how to handle
-  the thread where the error is generated, most likely by terminating it.
-
-See the current architecture implementations for examples.
+In the event of an unhandled CPU exception, the architecture code must call
+into :c:func:`z_fatal_error`. If the platform supports it, it may optionally
+implement :c:func:`z_arch_system_halt` to power off the system. The default
+implementation locks interrupts and spins forever.
+
+In the event of an unhandled CPU exception, the architecture
```

Comment:
Hmmm. My intention was to replace the previous paragraph with this one, deleting lines 444-448

Reply:
derp, fixed